### DISCUSSION TITLE
I don't have a dated School ID

### DISCUSSION THREAD

**Discussion by DanaiRo** [community/community#41616] [GitHub Education]
GitHub requested me for student verification --exclusively- a dated School ID. My uni doesn't provide a dated ID. Cld u plz give me suggestions?

**Comment by samus-aran** [comment]
Hi @DanaiRo 👋 

Although a dated School ID is the most preferred we do accept other forms of evidence. Here is a few tips on supplying alternative evidence.

1. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Dated Student ID being at the top. If you don't have a Student ID do try and acquire evidence that is positioned higher rather than lower in that option list.

2. Ensure your image (photo or upload) is eligible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

3. Ensure that your evidence corresponds with details on your GitHub profile. The verification process uses your GitHub username, email and profile name (and other aspects).

**Reply by DanaiRo** [reply]
Hello, thanks for the answer. But this doesn't work. Even though I upload my transcript pic the system asks me for my dated university id. I uploaded a screenshot. That seems like a big issue cause other  students I know face it as well. 
![B3B12C8B-428E-4163-A4AA-40105D35A1E5](https://user-images.githubusercontent.com/108496575/207921060-d2df2e52-fac7-4342-966c-caa552e698ae.png)

**Comment by lSamWorsnopl** [comment]
The same thing happens to me, although it used to let me upload documents for verification, but it doesn't let me anymore.
When taking a photo it always asks me for a dated school ID which I don't have.
![321494216_1385538168857441_1681143239503034570_n](https://user-images.githubusercontent.com/114540198/209417699-575b1d3c-eb27-4e2c-81e3-5c201cbe7d8e.jpg)

**Comment by silkyland** [comment]
From my experience  just add : "Valid until : Month/Year" to in the end of your proof.

**Comment by hasini7-tatikonda** [comment]
Same, Instead of my school ID, can I upload a picture from my student portal? It contains my college ID.
